### ISSUE TITLE
remote_execution_proxies shows the task changed every run

### ISSUE THREAD

**Issue by jmartensson** [RedHatSatellite/satellite-ansible-collection#16]
##### SUMMARY
Using `remote_execution_proxies` shows the task changed every run even if nothing is changed.

##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION

```
ansible 2.9.10
```

##### KATELLO/FOREMAN VERSION

```
tfm-rubygem-katello-3.14.0.21-1.el7sat.noarch
foreman-1.24.1.21-1.el7sat.noarch
```

##### APYPIE VERSION

```
0.2.2
```

##### STEPS TO REPRODUCE

Set the parameter `remote_execution_proxies`

```yaml
redhat.satellite.subnet:
  remote_execution_proxies:
  - server.example.com
  ...
```

##### EXPECTED RESULTS

ok: [localhost]

##### ACTUAL RESULTS

changed: [localhost]

```

```

**Comment by ehelms** [comment]
Please also report this bug to Bugzilla under the 'Ansible Collection' category for the 'Red Hat Satellite' product. That will help ensure it gets triaged.

**Comment by evgeni** [comment]
This is a known bug in the remote_execution plugin, will be fixed when Satellite includes version 4.1 of the plugin.

#17 contains a documentation update to refer to this. The upstream bug is: https://projects.theforeman.org/issues/30651

**Comment by evgeni** [comment]
closing this as 1.5.0 is released now
